mail/postfix/patches/300-bdb_hash_segfault.patch

```diff
@@ -1,10 +1,9 @@
-diff -Naur postfix-2.11.3/src/util/dict_db.c postfix-2.11.3_patched/src/util/dict_db.c
---- postfix-2.11.3/src/util/dict_db.c	2012-01-25 04:41:08.000000000 +0400
-+++ postfix-2.11.3_patched/src/util/dict_db.c	2014-11-01 12:36:44.287641712 +0300
-@@ -687,8 +687,8 @@
+--- a/src/util/dict_db.c	2017-07-16 16:56:00.819659962 +0300
```

Comment:
What happened here?  More than just the line numbers getting updated by a quilt refresh...